web/src/services/api/appearanceApi/utils.ts

```diff
@@ -0,0 +1,19 @@
+import { GetSceneQuery } from "../../gql";
+
+export type NLSAppearance = {
+  id: string;
+  name: string;
+  value?: any;
+};
+
+export const getAppearances = (rawScene?: GetSceneQuery) => {
+  const scene = rawScene?.node?.__typename === "Scene" ? rawScene.node : undefined;
+
+  return scene?.styles?.map((l): NLSAppearance => {
+    return {
+      id: l.id,
+      name: l.name,
+      value: l.value,
```

Comment:
if you will do single letters for params, please make them make sense. `l` is good for layer, but `a` is good for appearance. Or `s` for styles. But either way, we only use it 3 times, so writing it out fully is clearest and most readable 🙏🏻

Reply:
I'll do this.